fedimint-escrow-client/src/cli.rs

```diff
@@ -57,18 +57,11 @@ pub(crate) async fn handle_cli_command(
             // finalize_and_submit txns to lock ecash by underfunding
             // how to call this method?
             let (operation_id, out_point, escrow_id) = escrow
-                .buyer_txn(
-                    Amount::from_sat(cost),
-                    buyer,
-                    seller,
-                    arbiter,
-                    retreat_duration,
-                )
+                .buyer_txn(Amount::from_sat(cost), seller, arbiter, retreat_duration)
```

Comment:
This should probably be `escrow.create_scrow` to match what it does. `buyer_txn` is neither verb, nor very informative.